src/Build/Microsoft.Build.csproj

```diff
@@ -45,10 +45,8 @@
   </ItemGroup>
 
   <ItemGroup Condition="'$(TargetFrameworkIdentifier)' != '.NETFramework'">
-    <PackageReference Include="System.Diagnostics.TraceSource" />
     <PackageReference Include="System.Reflection.Metadata" />
-    <PackageReference Include="System.Reflection.TypeExtensions" />
-    <PackageReference Include="System.Runtime.Loader" />
+    <PackageReference Include="System.Runtime.Loader" Condition="'$(TargetFramework)' != 'netcoreapp2.1'" />
```

Comment:
Can you remove `System.Runtime.Loader` entirely? I don't think this condition will ever fire because this project is built only for `netcoreapp2.1` and `net472`.

Reply:
Done. With the same reasoning, I removed the condition on `System.Security.Principal.Windows`, as it would always fire.